css-canvas-medical/index.html

```diff
@@ -0,0 +1,121 @@
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container padding-top-bottom-15">
+    <div class="row">
+      <div class="column-half">
+        <img src="images/logo-blue.svg" alt="logo-blue">
+      </div>
+      <div class="column-half flex-end">
+        <div class="nav-bar">
+          <a>Product</a>
+          <a>Implementation</a>
+          <a>About Us</a>
+          <a>Blog</a>
+        </div>
+        <div class="contact-us-button">
+          <button>Contact Us</button>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="border-bottom margin-bottom-75"></div>
+  <div class="container margin-bottom-75">
+    <div class="row">
+      <div class="hero-image">
+        <div class="overlay"></div>
+        <div class="overlay-text">
+          <h1>The EMR Reimagined for Primary Care</h1>
+          <p>Can't remember the last time you talked with a patient without focusing on the EMR?
+            Canvas lets you rediscover joy in the work you do. Capture meaningful data, identify gaps in
+            care, and move seamlessly between tasks within the platform. You stay present with patients
+            while your whole care team stays up-to-speed. Be a better doctor with Canvas, designed for primary
+            care providers with the help of primary care providers.
+          </p>
+          <div class="get-started-button margin-top-40">
+            <button>Get Started</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="bg-FAFAFA text-align-center padding-75 color-6E6E6E">
+    <div class="container">
+      <div class="row">
+        <div class="column-full margin-left-right-auto">
+          <h2>Clinicians Prefer Canvas</h2>
+          <p>Physician owned primary care practices across the country have</p>
+          <p>already made Canvas the fabric of their care.</p>
+        </div>
+      </div>
+      <div class="row margin-top-40 line-height-40">
+        <div class="column-half">
+          <h3>Cascade Family Practice, Portland OR</h3>
+          <h3>Temescal Creek Medicine, Oakland CA</h3>
+          <h3>San Juan Healthcare, Friday Harbor WA</h3>
+        </div>
+        <div class="column-half">
+          <h3>Ward Medical Clinic, Bonifay FL</h3>
+          <h3>Caribou Medical, Towanda PA</h3>
+          <h3>Bay Tree Family Care, Red Bay AL</h3>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container padding-top-bottom-50">
+    <div class="row justify-space-between align-center color-6E6E6E">
+      <div>
+        <h2>Ready to learn more?</h2>
+        <p>Schedule a private meeting now</p>
+      </div>
+      <div class="get-started-button">
+        <button>Get Started</button>
+      </div>
+    </div>
+  </div>
+  <div class="bg-515351 padding-top-bottom-100">
+    <div class="row container align-center">
```

Comment:
One element will never be both a `container` and a `row`. `container` elements hold `row` elements as children.